python/lsst/sims/photUtils/Bandpass.py

```diff
@@ -441,6 +441,39 @@ def calcM5(self, skysed, hardware, expTime=PhotometricDefaults.exptime,
         Pass into this function the bandpass, hardware only of bandpass, and sky sed objects.
```

Comment:
These all seem to be the same parameters and same values as in catUtils.  Is it worth making a class to hold these parameters so we only have to define the defaults one place?